0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,92 @@
 ---
-name: CMX to CDP protocol
-shortname: 6/CMX2CDP
-status: raw
+name: Configuration Data Transport protocol
+shortname: 6/CDT
```

Comment:
Well, there is a subtle distinction between the *protocol name* (CDT) and the name of the *RFC that defines the protocol* (6/CDT).